src/Tile/index.js

```diff
@@ -0,0 +1,156 @@
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  border: solid 1px ${({ theme }) => theme.orbit.paletteCloudNormal}; //TODO Create token borderWidthTile, borderColorTile
+  box-shadow: 0 2px 4px 0 rgba(23, 27, 30, 0.1); //TODO Create token boxShadowTile
+  box-sizing: border-box;
+  transition: box-shadow ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+  font-family: ${({ theme }) => theme.orbit.fontFamily};
+  text-decoration: none;
+  overflow: hidden;
+  cursor: pointer;
+
+  &:hover,
+  &:focus {
+    outline: 0;
+    box-shadow: 0 4px 12px 0 rgba(23, 27, 30, 0.1); //TODO Create token boxShadowTileHover
+  }
+`;
+
+StyledTile.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledContent = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledHeadingWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  margin: ${({ theme }) => `0 0 ${theme.orbit.spaceXXSmall} 0`};
+  &:last-of-type {
+    margin: 0;
+  }
+`;
+StyledHeadingWrapper.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledChildren = styled.div`
+  font-size: ${({ theme }) => theme.orbit.fontSizeTextNormal};
+  color: ${({ theme }) => theme.orbit.colorTextPrimary};
+  line-height: ${({ theme }) => theme.orbit.lineHeightText};
+  margin: ${({ theme, icon, title }) => icon && title && `0 0 0 ${theme.orbit.spaceXLarge}`};
+`;
+
+StyledChildren.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledIcon = styled.div`
+  color: ${({ theme }) => theme.orbit.colorHeading};
+  display: flex;
+  align-items: center;
+  margin: ${({ theme }) => `0 ${theme.orbit.spaceXSmall} 0 0`};
+`;
+
+StyledIcon.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledIconRight = styled.div`
+  display: flex;
+  flex-shrink: 0;
+  padding: ${({ theme }) => `0 0 0 ${theme.orbit.spaceMedium}`};
+`;
+
+StyledIconRight.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledNewWindow = styled(NewWindow)`
+  display: flex;
+  align-items: center;
+  color: ${({ theme }) => theme.orbit.paletteInkLight};
+  transition: color ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+
+  &:hover {
+    color: ${({ theme }) => theme.orbit.paletteInkLightHover};
+  }
+`;
+
+StyledNewWindow.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledChevron = styled(ChevronRight)`
+  display: flex;
+  align-items: center;
+  color: ${({ theme }) => theme.orbit.paletteInkLight};
+  transition: color ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+
+  &:hover {
+    color: ${({ theme }) => theme.orbit.paletteInkLightHover};
+  }
+`;
+
+StyledChevron.defaultProps = {
+  theme: defaultTokens,
+};
+
+const Icon = ({ icon }: Props) => <StyledIcon>{icon}</StyledIcon>;
+
+const Tile = (props: Props) => {
+  const { children, external = false, href, icon, title } = props;
+  return (
+    <StyledTile target={href && external ? "_blank" : undefined} href={href} {...props}>
```

Comment:
There are any other ...props ? If yes, please destruct it for better transparency